core/versioning/txVersionChecker.go

```diff
@@ -5,9 +5,11 @@ import (
 	"github.com/ElrondNetwork/elrond-go-core/data/transaction"
 )
 
-const (
-	initialVersionOfTransaction = uint32(1)
-)
+// There are different options based on the tx version:
```

Comment:
can you simply explain the flags not their combination?
if we add several more flags, the description of their combination will be too long

options: 
bit0: signed over hash - if set, the transaction signature was applied over the hash of the marshalled tx, otherwise directly on the marshalled tx
bit1: guarded Tx - if set, the transaction is guarded (co signed) by the sender's configured guardian which should be the same as the one configured in the Tx GuardianAddr field.

Reply:
:+1:

Reply:
updated, thanks